docs/guides/modules/test/pages/adaptive-testing.adoc

```diff
@@ -776,6 +796,37 @@ Yes! The branch behavior is fully customizable through your CircleCI configurati
 
 See Scenario 3 in the "Flag Usage Scenarios" section for examples of customizing branch behavior.
 
+=== Can I run test analysis and selection on any branch?
+
+Yes! The branch behavior is fully customizable through your CircleCI configuration. While analysis runs on main by default, you can configure it to run on:
+
+. Any specific branch (for example, `develop` or `staging`).
+. Feature branches if needed for testing.
+. Scheduled pipelines.
+
+See Scenario 3 in the "Flag Usage Scenarios" section for examples of customizing branch behavior.
+
+[#baseline-coverage]
+=== Why are there so many files impact a test?
+
+If you see many files impacting each test during analysis (e.g., "...found 150 files impacting test..."), this may be caused by shared setup code like global imports or framework initialization being included in coverage.
```

Comment:
```suggestion
If you see many files impacting each test during analysis (for example, "...found 150 files impacting test..."), this may be caused by shared setup code like global imports or framework initialization being included in coverage.
```